modules/apps/transfer/keeper/msg_server.go

```diff
@@ -62,7 +61,7 @@ func (k Keeper) Transfer(goCtx context.Context, msg *types.MsgTransfer) (*types.
 			types.EventTypeTransfer,
 			sdk.NewAttribute(types.AttributeKeySender, msg.Sender),
 			sdk.NewAttribute(types.AttributeKeyReceiver, msg.Receiver),
-			sdk.NewAttribute(types.AttributeKeyTokens, tokens.String()),
+			sdk.NewAttribute(types.AttributeKeyTokens, coins.String()),
```

Comment:
this is already an existing issue that on send we emit a different coin type than on ack (send uses coin denom while ack uses token denom (path))

In a followup pr, we should probably move the event emission into an event.go file and emit based on the tokens sent in the packet data

same for logger